modules/util/source-formatter/src/main/resources/dependencies/closeable-type-names.json

```diff
@@ -30,6 +30,9 @@
 		{
 			"name": "com.github.scribejava.core.oauth.OAuth20Service"
 		},
+		{
```

Comment:
@hhuijser is this ok for Simon to add in this way?  Or should he send a separate pull?

Reply:
this is not manually added. I will remove it.